HighscoreWindow.cpp

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright 2022, Harshit Sharma <harshits908@gmail.com>
+ * This file is distributed under the terms of the MIT license
+ */
+
+#include "HighscoreWindow.h"
+
+#include "WindowBoard.h"
+
+#include <Catalog.h>
+#include <string>
+#include <LayoutBuilder.h>
+#include <Message.h>
+#include <String.h>
+#include <StringView.h>
+
+#undef B_TRANSLATION_CONTEXT
+#define B_TRANSLATION_CONTEXT "HighscoreWindow"
+
+HighscoreWindow::HighscoreWindow(const char* oldHighscorer, const int32 oldHighscore, const int32 newHighscore)
+	:
+	BWindow(BRect(200, 200, 650, 600), B_TRANSLATE_SYSTEM_NAME("Enter you High Score"), B_TITLED_WINDOW, 0)
```

Comment:
B_TRANSLATE_SYSTEM_NAME should be used for the application name instead (this makes it available for showing in DeskBar). For other strings just use B_TRANSLATE